feature/achievements/src/main/java/io/github/droidkaigi/confsched2023/achievements/AchievementsScreen.kt

```diff
@@ -83,51 +90,85 @@ fun AchievementsScreen(
         snackbarHostState = snackbarHostState,
         contentPadding = contentPadding,
         onReset = viewModel::onReset,
+        showAnimation = { achievement -> viewModel.onClickAchievement(achievement) },
+        finishAnimation = viewModel::onFinishAnimation,
         onDisplayedInitialDialog = viewModel::onDisplayedInitialDialog,
     )
 }
 
 data class AchievementsScreenUiState(
     val achievementListUiState: AchievementListUiState,
     val isShowInitialDialog: Boolean,
+    val clickedAchievement: ClickedAchievementState,
 )
 
+sealed interface ClickedAchievementState {
```

Comment:
How about naming this AchievementAnimationState and AchievementAnimationState.Animating AchievementAnimationState.NotAnimating?

Reply:
The namings look better. I will rename them.